tpot/gp_deap.py

```diff
@@ -149,6 +149,17 @@ def varOr(population, toolbox, lambda_, cxpb, mutpb):
 
     return offspring
 
+def initialize_stats_dict(individual):
+    '''
+    Initializes the stats dict for individual
+    :param individual:
+    :return:
```

Comment:
Please make sure the function docstrings match the docstring format that we use in the rest of TPOT.

Reply:
Will do. Is this any established format, then i can set my IDE settings which will save me some time.

Reply:
Moreover, there is some discrepancy in the format used. E.g. comparing mutate_random_individual and varOr in gp_deap.